### DISCUSSION TITLE
Frigate object detector accessible from external Python programs?

### DISCUSSION THREAD

**Discussion by 0ip** [blakeblackshear/frigate#8471] [Ask A Question] [answered]
Over the years, I've built my own home automation stack or home assistant clone of sorts.
I want to integrate Frigate's object/person detection into my own Python-based backend.
Does Frigate provide simple interfaces of its detection architecture to external software like `frigate.detect_objects(video_path)`?

**Comment by NickM-27** [comment, marked as answer]
No, frigate doesn't offer on demand processing of existing video files. You can configure the config to read from a local file but it is a manual process.

**Reply by 0ip** [reply]
Got it, thank you for clarifying!
